slides/src/content/unit-testing.md

```diff
@@ -332,39 +275,63 @@ beforeEach(() => {
 ```
 
 ---
-<!-- .slide: id="unit-testing-change-detection-1" -->
-## Change Detection
 
-- We must tell Angular when to run change detection during our tests
-- Use the fixture to do this:
+<!-- .slide: id="unit-testing-shallow-rendering" -->
+## Shallow Rendering
+
+- The above test fails because `AppComponent` itself uses components which we have not declared.
+- We could provide mocks for these components, or we can do a shallow render by adding the `CUSTOM-ELEMENTS-SCHEMA`
 
 #####_src/app/app.component.spec.ts_
 ```ts
-fixture = TestBed.createComponent(AppComponent);
-fixture.detectChanges()
+TestBed.configureTestingModule({
+  schemas: [ CUSTOM_ELEMENTS_SCHEMA ]
+})
 ```
 
+Notes:
+
+- `NO_ERRORS_SCHEMA` is also commonly used for shallow rendering.
```

Comment:
1.  Remove "Notes:"
2.  We haven't introduced schemas previously - add a slide explaining what they are?